adele-booking/src/main/java/com/microservicesteam/adele/booking/boundary/web/BookingController.java

```diff
@@ -25,8 +26,8 @@ public BookingController(BookingService bookingService) {
     }
 
     @GetMapping
-    public List<Ticket> getBookings(@RequestParam long eventId) {
-        return bookingService.getTicketsStatus(eventId);
+    public List<Ticket> getBookings(@RequestParam long eventId, @RequestParam(required = false) Integer sectorId) {
+        return bookingService.getTicketsStatus(eventId, Optional.ofNullable(sectorId));
```

Comment:
Some says that Optional should not be parameter or field but return type only.
How about creating another overload for `getTicketsStatus` and see how the result looks like?
(`getTicketsStatusByEventAndSector` etc...)